app/messaging/index.js

```diff
@@ -4,15 +4,21 @@ const { MessageReceiver } = require("adp-messaging");
 let plpReceiver;
 
 async function start() {
-  const plpAction = (message) => processPlpMessage(message, plpReceiver);
-  plpReceiver = new MessageReceiver(config.plpSubscription, plpAction);
-  await plpReceiver.subscribe();
+  // test if not running locally
+  if (config.plpSubscription.name) {
```

Comment:
Hi David, on Friday, we agreed not to perform this test as Production should blow-up if the name isn't set. At the moment, it won't. Granted there will be no "Ready" message but no error either. Can you please remove the check? Thank you in advance. Jason